mozilla-mobile/firefox-android
# Bug 1807126 - Increased gap between slider and scale value

### What
The dot from the scale overlaps the percentage number on Accessibility screen

### Fix description
Increased seekBar end margin from 60 to 70.

### Steps to test

1. From the device's settings, change the font size to the biggest option (extra large/huge).
2. Tap on 3 dot menu.
3. Open Settings.
4. Go to Accessibility and change the scale to a maximum (200%).
5. Observe the page.

### Screenshots
Issue video:
https://www.loom.com/share/23e1da8882fd48179c16b6a5ee2c353f

Fix video:
https://www.loom.com/share/59a4cf570c09455b845f74f290bc4b64

### Pull Request checklist
 - [ ] Changelog: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
 - [ ] Accessibility: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features

### After merge
- Milestone: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- Breaking Changes: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

---
This code was written and reviewed by GitStart Community. Growing future engineers, one PR at a time.




### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1807126
